cfpb/cfgov-refresh
# Mega menu: correct width of mega menu horizontal rule and responsive sizes

Supersedes https://github.com/cfpb/cfgov-refresh/pull/5557

## Changes

- Adjusts width and relative positioning of desktop menu.

## Testing

1. Pull down and build.
3. View at 1261px or greater.
4. Open the menu and check that as it animates down that the horizontal gray rule fully extends to the end of the menu on the left and right side (open one of the center menu items, such as data & research, to make this easier to see). Also note that the gap on the left and right sides of the menu and the edge of the window are the same width.
2. Resize your browser width to 1000px and repeat the above and compare consumerfinance.gov to localhost.

## Screenshots

<img width="1491" alt="Screen Shot 2020-03-10 at 11 54 56 AM" src="https://user-images.githubusercontent.com/704760/76331904-15fa6700-62c6-11ea-9186-24ab887e6e20.png">

<img width="1249" alt="Screen Shot 2020-03-10 at 11 55 31 AM" src="https://user-images.githubusercontent.com/704760/76331909-172b9400-62c6-11ea-9320-ccc444b7d681.png">

## Todo

We need to break from the xtra-large desktop size to the desktop size earlier as there is a small window where the horizontal scrollbar appears.
